mcsema/cfgToLLVM/JumpTables.cpp

```diff
@@ -229,7 +229,17 @@ static void doJumpTableViaSwitch(TranslationContext &ctx, llvm::BasicBlock *&blo
   int myindex = 0;
   for (auto jmp_block_va : jmpblocks) {
     BasicBlock *toBlock = ctx.va_to_bb[jmp_block_va];
-    TASSERT(toBlock != NULL, "Could not find block");
+     
+    // FIXME: this needs more testing, I (Josh) added it and I'm not 100% on the syntax, but it's worked thus far
+    if (!toBlock)
+    {
+       std::cerr << "WARNING: NULL block found... adding unreachable instruction" << std::endl;
```

Comment:
Add in some context telling where the block is using `std::hex << jmp_block_va`.